### DISCUSSION TITLE
Mail merge range

### DISCUSSION THREAD

**Discussion by aclermont123** [plainlab/plainmerge#26] [Q&A] [answered]
Hi,
Is there a way to generate the mail merge to start at line lets say 500 instead of 1 
or 
mailmerge 500 to 1000 instead of 1 to 1000?

Thanks

**Comment by manhtai** [comment, marked as answer]
Hi,

There isn’t a feature like that for now but you can always trim the Excel file yourself before merging.

Cheers!
